web/src/beta/lib/core/Map/ClusteredLayers/index.tsx

```diff
@@ -15,6 +15,7 @@ export type Props = {
   clusters?: Cluster[];
   delegatedDataTypes?: DataType[];
   sceneProperty?: any;
+  showStoryPanel?: boolean;
```

Comment:
Can we remove this as well?

Reply:
@keiya01 this one is used to check the selected tab.For the story page we need to display layers based on the page, so if I not check which tab is selected, then when navigate to other tab it will show the layers based on  previous selected page. so showStoryPanel is need.